MAUI/Calendar/events.md

```diff
@@ -179,3 +179,236 @@ private void OnCalendarLongPressed(object sender, CalendarLongPressedEventArgs e
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="3,6,8" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                     DoubleTappedCommand="DoubleTappedCommand">
+<ContentPage.BindingContext>
+    <local:CalendarViewModel/>
+</ContentPage.BindingContext>					  
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="1" %}
+
+public class CalendarViewModel
+{
+    public ICommand DoubleTappedCommand { get; set; }
+    public CalendarViewModel()
+    {
+        DoubleTappedCommand = new Command<CalendarDoubleTappedEventArgs>(DoubleTapped);
+    }
+    private void DoubleTapped(CalendarDoubleTappedEventArgs args)
+    {
+        // To do your requirement here.
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### LongPressedCommand
+
+SfCalendar includes a built-in event called `LongPressed` and invokes this event through the [LongPressedCommand]() whenever the calendar view is long pressed, with the `CalendarLongPressedEventArgs` passed as parameter.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                     LongPressedCommand="LongPressedCommand">
+<ContentPage.BindingContext>
+    <local:CalendarViewModel/>
+</ContentPage.BindingContext>					  
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="3,6,8" %}
+
+public class CalendarViewModel
+{
+    public ICommand LongPressedCommand { get; set; }
+    public CalendarViewModel()
+    {
+        LongPressedCommand = new Command<CalendarLongPressedEventArgs>(LongPressed);
+    }
+    private void LongPressed(CalendarLongPressedEventArgs args)
+    {
+        // To do your requirement here.
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### AcceptCommand
+
+SfCalendar includes a built-in event called `ActionButtonClicked` and invokes this event through the [AcceptCommand]() whenever confirm button tapped on calendar.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="2" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                     AcceptCommand="AcceptCommand">
+<ContentPage.BindingContext>
+    <local:CalendarViewModel/>
+</ContentPage.BindingContext>					  
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="3,6,8" %}
+
+public class CalendarViewModel
+{
+    public ICommand AcceptCommand { get; set; }
+    public CalendarViewModel()
+    {
+        AcceptCommand = new Command(ActionButtonClicked);
+    }
+    private void ActionButtonClicked()
+    {
+        // To do your requirement here.
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+### DeclineCommand
+
+SfCalendar includes a built-in event called `ActionButtonCanceled` and invokes this event through the [DeclineCommand]() whenever the cancel button tapped on calendar.
+
```

Comment:
The `SfCalendar` includes a built-in event called `ActionButtonCanceled`, which is triggered when the cancel button is tapped on the calendar. This event can be invoked through the [DeclineCommand]().

Reply:
Content updated.